### ISSUE TITLE
lastversion does not correctly fetch a version from github releases

### ISSUE THREAD

**Issue by voriol** [dvershinin/lastversion#109]
lastversion does not correctly fetch a version from github releases

It is strange, because in the case of the kibana software, elastic does not correctly find the version

```
$ lastversion elastic/kibana
1694506029

$ lastversion https://github.com/elastic/kibana/releases
1694506029
```

In this case, the correct answer is: [8.10.0](https://github.com/elastic/kibana/releases/tag/v8.10.0)

When you have no problems with the rest of the products:

```
$ lastversion elastic/elasticsearch
$ lastversion elastic/beats
$ lastversion elastic/logstash
```

**Comment by dvershinin** [comment]
Added fix for this in the newly released lastversion v3.1.1

**Comment by voriol** [comment]
Awsome, thanks @dvershinin
